src/components/Header.astro

```diff
@@ -21,11 +21,9 @@ import DevelopersLogo from "../components/DevelopersLogo.astro";
         Interledger Foundation
       </span>
   </a>
-  <div class="secondary-wrap">
+  <div class="sl-hidden md:sl-flex secondary-wrap">
```

Comment:
Bit of a bug fix, because the Github icon shows up in the mobile menu, and should have been part of the secondary wrap to begin with.